src/bmi_troute.py

```diff
@@ -370,7 +386,23 @@ def get_value_ptr(self, var_name):
         array_like
             Value array.
         """
-        return self._values[var_name]
+        if var_name.endswith("__count"):
```

Comment:
I think it's maybe preferable to just not support `get_value` and `get_value_ptr` for the `foo__count` variables, rather than synthesizing a pointer to a non-persistent object.